metrotranscom/doorway
# feat: creates script runner and translations for new reserved community type

## Issue Overview

This PR addresses [#issue](https://github.com/metrotranscom/doorway/issues/660)


## Description
Creates a new script runner entry to create a reserved community type, and includes the new translations

## How Can This Be Tested/Reviewed?
hit the script runner endpoint passing an id dto of the jurisdiction you want to add the community type too

log into the partner site and add the new community type to a listing
go to the public site and make sure it comes through properly (and is translated correctly)

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
